app/views/admin/legislation/draft_versions/_form.html.erb

```diff
@@ -14,14 +16,17 @@
   <% end %>
 
   <div class="small-12 medium-9 column">
-    <%= f.label :title %>
-    <%= f.text_field :title, label: false, placeholder: t("admin.legislation.draft_versions.form.title_placeholder") %>
+    <%= f.translatable_text_field :title,
+                                  placeholder: t("admin.legislation.draft_versions.form.title_placeholder") %>
   </div>
 
   <div class="small-12 medium-9 column">
     <%= f.label :changelog %>
     <span class="help-text"><%= t("admin.legislation.draft_versions.form.use_markdown") %></span>
-    <%= f.text_area :changelog, label: false, rows: 5, placeholder: t("admin.legislation.draft_versions.form.changelog_placeholder") %>
+    <%= f.translatable_text_area :changelog,
+                                 label: false,
```

Comment:
@javierm Here and in a few other fields which have a help-text, I've left the explicitly-defined label, instead of relying on the field helper to generate the label. This is so that we could have the help-text tag _between_ the label and input elements. 

This has the disadvantage of that we don't have one label per locale. I guess we could change the `TranslatableFormBuilder#translatable_text_field` helper to accommodate a help-text tag (generate one pre locale). Do you think it's worth it?

Reply:
Personally I'd probably add an option similar to [simple form](https://github.com/plataformatec/simple_form)'s `hint`, but I'm not sure it's worth it in this case. Whatever you prefer is fine with me.